apps/generator/lib/generator.js

```diff
@@ -967,19 +967,34 @@ class Generator {
    * @private
    */
   async loadTemplateConfig() {
```

Comment:
please pull it outside `generator.js`, just like we have for example template config validation in [templateConfigValidator.js](https://github.com/asyncapi/generator/blob/master/apps/generator/lib/templateConfigValidator.js)

with each PR touching we should try to move code outside this gigantic `generator.js`

maybe we need new folder? like templateConfig,  where we will have `validator.js` and `loader.js`

or maybe this should be a follow-up to cleanup?